pkg/controller/shared/shared_informer.go

```diff
@@ -81,13 +87,26 @@ type sharedInformerFactory struct {
 	customListerWatchers  ListerWatcherOverrides
 	defaultResync         time.Duration
 
+	templateInformers      templateinformers.SharedInformerFactory
```

Comment:
What do you want the tsruct to look like?  The upstream shared struct?  This is the generated type